pkgs/tools/typesetting/tex/texlive/default.nix

```diff
@@ -155,13 +154,24 @@ let
     xzcat "$tlpdbxz" | sed -rn -f "$tl2nix" | uniq > "$out"
   '';
 
+  # map: name -> fixed-output hash
+  fixedHashes = lib.optionalAttrs useFixedHashes (import ./fixed-hashes.nix);
+
+  # NOTE: the fixed naming scheme must match generated-fixed-hashes.nix
+  # name for the URL
+  mkURLName = { pname, tlType, ... }: pname + lib.optionalString (tlType != "run" && tlType != "tlpkg") ".${tlType}";
+  # name + revision for the fixed output hashes
+  mkFixedName = { revision, extraRevision ? "", ... }@attrs: mkURLName attrs + ".r${toString revision}${extraRevision}";
```

Comment:
this picks the same name for "run" and for "tlpkg" packages, which ultimately results in the tlpkg packages being replaced by dupicates of the run packages.

Reply:
Oh my, I forgot the `tlpkg` in all the *other* relevant places. Fixed now, I hope (they popped up in the new `fixed-hashes.nix` and I get exceptions if I omit one, so it should be good).